tests/unit/PushTest.php

```diff
@@ -448,6 +456,62 @@ public function hasValidMappedPpPaProperties()
         );
     }
 
+    /**
+     * @test
+     *
+     * @throws XmlResponseParserException
+     */
+    public function hasValidMappedCompanyProperties()
+    {
+        $push = new Push($this->xmlSampleCompanyResponse);
+        $response = $push->getResponse();
+
+        if (!($response instanceof Response)) {
+            throw new \RuntimeException('Response is not set!');
+        }
+
+        $this->assertEquals('heidelpay GmbH', $response->getCompany()->companyname);
```

Comment:
Consider local variables to avoid calling the getters this often.

```suggestion
        $company = $response->getCompany();
        $this->assertEquals('heidelpay GmbH', $company()->companyname);
```

Reply:
done.